index.html

```diff
@@ -31,14 +31,164 @@ <h1 class="mainTitle">I’m Damian.
+                    </ul>
+                    <button class="seeProjectButton">See project</button>
+                </div>
+                <div></div>
+            </div>
+            <div class="card">
+                <div class="pictureContainer"><img class="pictures" src="./img/cards/Pic4.svg" alt="Tonic"></div>
+                <div class="cardContent">
+                    <h3 class="cardsTitle">Tonic</h3>
+                    <ul class="subTitleList">
+                        <li class="canopy">CANOPY</li>
+                        <li><img class="dots" src="./img/cards/dot.svg" alt="dot"></li>
+                        <li class="grayWords">Back End Dev</li>
+                        <li><img class="dots" src="./img/cards/dot.svg" alt="dot"></li>
+                        <li class="grayWords">2015</li>
+                    </ul>
+                    <p class="pInfo">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required.</p>
+                    <ul class="techList">
+                        <li class="techs">HTML</li>
+                        <li class="techs">CSS</li>
+                        <li class="techs">JavaScript</li>
+                    </ul>
+                    <button class="seeProjectButton">See project</button>
+                </div>
+                <div></div>
+            </div>
+            <section class="about">
+                <h2 class="mainTitle">About Myself</h2>
+                <p class="aboutP">I’m a software developer! I can help you build a product , feature or website Look
+                    through some of my
+                    work
+                    and experience! If you like what you see and have a project you need coded, don’t hesitate to
+                    contact
+                    me.
+                </p>
+
+                <h3>LET’S CONNECT</h3>
+                <ul id="iconListContainer">
+                    <li><a href="https://github.com/DamianDib9"><img class="iconList" src="./img/icon/guthub.svg"
+                                alt="github"></a></li>
+                    <li><a href="https://www.linkedin.com/in/damian-dib-bustros-953577261/"><img class="iconList"
+                                src="./img/icon/linkedin.svg" alt="Linkedin"></a></li>
+                    <li><a href="https://www.angellist.com/"><img class="iconList" src="./img/icon/peace.svg"
+                                alt="Peace"></a></li>
+                    <li><a href="https://twitter.com/DevBustros"><img class="iconList" src="./img/icon/twitter.svg"
+                                alt="Twitter"></a></li>
+
+                    <li><a href="https://www.google.com/"><img class="iconList" src="./img/icon/mlogo.svg"
+                                alt="mlogo"></a>
+                    </li>
+
+
+                </ul>
+                <button class="seeProjectButton">Get my resume</button>
+
+                <div class="languages">
```

Comment:
- [ ] According to project requirements, this should be a ul tag with it's children being an li tag also. see example below:

```
                <ul class="languages">
                    <div class="techTitles">
                        <h4>Languages</h5> <img src="./img/tech logos/arrow down.svg" alt="arrow">
                    </div>
                    <li>
                    <ul class="desplTechList">
                        <li class="techDescr"><img class="logoPadding" src="./img/tech logos/js.svg" alt="JavasCript">
                            <h5 class="techText">JavaScript</h5>
                        </li>
                        <li class="techDescr"><img class="logoPadding" src="./img/tech logos/html.svg" alt="HTML">
                            <h5 class="techText">HTML</h5>
                        </li>
                        <li class="techDescr"><img class="logoPadding" src="./img/tech logos/css.svg" alt="CSS">
                            <h5 class="techText">CSS</h5>
                        </li>
                      </ul>
                    </li>
                    <li>
                        <ul class="techTitles" class="underline">
                            <h4>Frameworks</h5> <img src="./img/tech logos/arrow right.svg" alt="arrow">
                        </ul>
                    </li>
                    <li>
                        <ul class="techTitles" class="underline">
                            <h4 >Skills</h5> <img src="./img/tech logos/arrow right.svg" alt="arrpw">
                        </ul>
                    </li>
                </ul>
```

Reply:
OK! Done